job-dsl-core/src/main/docs/examples/javaposse/jobdsl/dsl/DslFactory/workflowMultiBranchJob.groovy

```diff
@@ -0,0 +1,17 @@
+workflowMultiBranchJob('example') {
+    sources {
+        branchSource {
```

Comment:
this should be called `git` since it creates a Git source. Try to use the UI labels for naming things.